clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/bastion/BastionCredentialsProvider.groovy

```diff
@@ -83,4 +85,23 @@ class BastionCredentialsProvider implements AWSCredentialsProvider {
     expiration = format.parse(json.Expiration as String)
     new BasicSessionCredentials(json.AccessKeyId as String, json.SecretAccessKey as String, json.Token as String)
   }
+
+  static class VerboseJSchLogger implements com.jcraft.jsch.Logger {
+    static Hashtable name = ImmutableMap.of(
+      new Integer(com.jcraft.jsch.Logger.DEBUG), "JSCH DEBUG: ",
+      new Integer(com.jcraft.jsch.Logger.INFO),  "JSCH INFO: ",
+      new Integer(com.jcraft.jsch.Logger.WARN),  "JSCH WARN: ",
+      new Integer(com.jcraft.jsch.Logger.ERROR), "JSCH ERROR: ",
+      new Integer(com.jcraft.jsch.Logger.FATAL), "JSCH FATAL: ",
+    )
+
+    boolean isEnabled(int level) {
+      return true
+    }
+
+    void log(int level, String message) {
+      System.err.print(name.get(level))
```

Comment:
System.err what what?

Seems like we could just `@Slf4j` and `log.debug` the message ... it doesn't seem all that important to worry about level.

Reply:
It did not actually help, let's close this